### PR TITLE
 Added axis name for unnamed Series visualization

### PR DESCRIPTION
**Pull request by caitlynachen**
For histograms with no column name attribute, use "Series" as a placeholder.
" " -> "Series (binned)"

Screen Shot 2021-02-11 at 10 19 15 PM
Screen Shot 2021-02-11 at 10 19 05 PM
https://app.zenhub.com/workspaces/lux-board-6000bc7686a00400181b23f3/issues/lux-org/lux/241

### CODE REVIEW

**Review by dorisjlee — APPROVED** [review]

**Comment by dorisjlee** [comment]
Thanks @caitlynachen ! Could you add a test case for this?

File: tests/test_vis.py [hunk 1]
```diff
@@ -497,3 +497,18 @@ def test_matplotlib_heatmap_flag_config():
     assert not df.recommendation["Correlation"][0]._postbin
     lux.config.heatmap = True
     lux.config.plotting_backend = "vegalite"
+
+
+def test_unnamed_column():
+    lux.config.plotting_backend = "matplotlib"
+    df = pd.read_csv("lux/data/car.csv")
+    df.plot_config = None
+    df = df.rename(columns={"Horsepower": " "})
```
Comment (dorisjlee) [inline tests/test_vis.py:506]: Can we change this line to the original example to make it more realistic? 
```python
df = pd.read_csv("https://raw.githubusercontent.com/lux-org/lux-datasets/master/data/employee.csv")
df["YearsAtCompany"]/df["TotalWorkingYears"]
```

**Review by dorisjlee — APPROVED** [review]

**Review by dorisjlee — APPROVED** [review]

File: lux/vislib/matplotlib/Histogram.py [hunk 2]
```diff
@@ -59,12 +59,15 @@ def initialize_chart(self):
 
         x_label = ""
         y_label = ""
+        axis_title = f"{msr_attr.attribute} (binned)"
```
Comment (westernguy2) [inline lux/vislib/matplotlib/Histogram.py:62]: Should we be using `msr_attr_abv = str(msr_attr_attribute)` like we do in Altair, or is that unnecessary?

**Review by westernguy2 — APPROVED** [review]

File: tests/test_vis.py (hunk 1 above)
Comment (caitlynachen) [reply tests/test_vis.py:506]: im not exactly sure how to get the vis.code for a LuxSeries object. is there a way to do this?

File: lux/vislib/matplotlib/Histogram.py (hunk 2 above)
Comment (caitlynachen) [reply lux/vislib/matplotlib/Histogram.py:62]: good catch!

File: tests/test_vis.py (hunk 1 above)
Comment (caitlynachen) [reply tests/test_vis.py]: fixed my adding recommendation attribute to luxSeries https://github.com/lux-org/lux/pull/266

**Comment by caitlynachen** [comment]
should be built on top of https://github.com/lux-org/lux/pull/266

**Pull request merged**